helpers/authentication.js

```diff
@@ -175,15 +170,12 @@ const login = (payload = {}, req, res, next) => {
 	const { redirect } = payload;
 	delete payload.redirect;
 	return api(req).post('/authentication', { json: payload }).then((data) => {
-		res.cookie('jwt', data.accessToken,
-			Object.assign({},
-				{
+		res.cookie('jwt', data.accessToken, {
 					expires: new Date(Date.now() + 30 * 24 * 60 * 60 * 1000),
 					httpOnly: false, // can't be set to true with nuxt client
 					hostOnly: true,
 					secure: NODE_ENV === 'production',
-				},
-				cookieDomain(res)));
+				});
```

Comment:
Vorher wurde hier wenn res.locals.domain in production gesetzt war das domain: xxx rein gemerged. Was hat das den gemacht?

Reply:
darum gehts ja grsde dass das raus fliegt. wenn man eine domain setzt, gilt das für alle subdomains und dann redirected der superhero oder open/brandenburg endloserweise...

Reply:
ok